app/helpers/kadmin/charts_helper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Kadmin
+  # Provide helpers for displaying charts based on http://www.chartjs.org/
+  # Eventually add more, and just map the labels/data correctly.
+  module ChartsHelper
+    def charts_doughnut_tag(*args)
+      charts_tag('doughnut', *args)
+    end
+
+    def charts_pie_tag(*args)
+      charts_tag('pie', *args)
+    end
+
+    def charts_tag(type, data = [], labels = [], html_options = {})
```

Comment:
Even though it's pretty straightforward to use, maybe add an example on how to use the tag (or referring to the tests which have an example).